app/views/support_users/feedbacks/general.html.slim

```diff
@@ -1,5 +1,9 @@
 - content_for :page_title_prefix, t(".title")
 
+- content_for :breadcrumbs do
+  nav.govuk-breadcrumbs aria-label="Breadcrumbs"
+  = govuk_back_link text: t("buttons.back"), href: support_user_root_path
+
```

Comment:
Could this also be added to the `feedbacks/job_alerts.html.slim` and `feedbacks/satisfaction_ratings.html.slim` (the other 2 tabs besides "general") ?